src/pages/dashboard.tsx

```diff
@@ -0,0 +1,60 @@
+import React, {useState, useEffect, Component} from 'react';
+import {useRouter} from 'next/router';
+
+import {Navbar, Page, Footer, GamerCard} from './../components/AppComponents';
+import {
+    Container,
+    Header,
+    CardBody,
+    Card,
+    Alert,
+    CardFooter,
+    Button,
+    Paragraph,
+    Form,
+    CardHeader,
+    Box,
+    Text,
+    Small,
+    Main,
+    LineBreak, UnorderedList, ListItem
+} from './../components/SimpleComponents';
+import {Input} from './../components/SmartComponents';
+
+import CONSTANTS from './../config/CONSTANTS';
+
+import {UserService} from './../components/Users';
+import UtilityService, {getBaseUrlWithProtocol} from '../services/UtilityService';
+import TypeService from '../services/TypeService';
+
+//===----=---=-=--=--===--=-===----=---=-=--=--===--=-===----=---=-=--=--===--=-//
+
+
+let DashboardPage = ({baseUrl}) => {
+
+    let router = useRouter();
+    let user = UserService.getUser();
+    console.log(user);
```

Comment:
There's got to be a way that we can validate and ensure that the user is always logged in before this page renders, cause it'd be annoying to have to always check "user && user.first_name" but I'm not sure how next.js does it.